benchmarks/coordinates.py

```diff
@@ -1,4 +1,5 @@
 import numpy as np
+import astropy.coordinates as coord
 from astropy.coordinates import SkyCoord, FK5, Latitude, Angle
```

Comment:
Why not just import ICRS on line 3?

Reply:
Well, I need `concatenate` too. But it just seemed like we were approaching manually typing out `from astropy.coordinates import *` :)